src/components/ec-inline-actions/ec-inline-actions.vue

```diff
@@ -74,61 +77,78 @@ export default {
       default: () => ({ offset: 10, placement: 'bottom-start' }),
     },
   },
+  methods: {
+    componentTag(item) {
+      if (item.href) {
+        return 'a';
+      }
+      return 'button';
+    },
+    getDownloadText(item) {
+      const hasHrefAndDowload = !!item.href && !!item.download;
+      if (hasHrefAndDowload) {
+        return item.download;
+      }
+      if (item.href) {
+        return '';
```

Comment:
This will make every anchor downloadable. You cannot make just plain anchor without download attribute. I'd do this only if download is set to an empty string, otherwise null.

Please write a test for it.